### ISSUE TITLE
Electrum-LTC won't start & icon sizes

### ISSUE THREAD

**Issue by LF76** [estevaocm/bitkey#16]
Thanks a lot for the fast updates, Steven! I just grabbed the iso to check the improvements.

I just wish to point out 2 bugs, maybe, if you can check them.

With the new icons in the wbar, it gets 'cut off' in some monitors, depending on the screen resolution.

For example, in a 1024x768 monitor, I can't click the icons for Electrum and Electrum-LTC, on the left size.

Maybe one way to fix it could be to reduce the icon size in the _wbar-config_ to 36, or 24. Or automatically according to the screen res.

I tried to use in the terminal 'xrandr --output LVDS --auto' as a workaround.

Also, when I start Electrum-LTC, nothing happens after I type the password. Testing in the terminal, there's this error: _No module named 'dns'. Try sudo pip install_...

Could you check it, please? Don't know if it's a package missing or a bug in the latest LTC version.

Thanks again for your time and help with the project.

**Comment by estevaocm** [comment]
Valeu, Luis Flavio!

Em inglês pros gringos poderem acompanhar:

I'll look into these issues as soon as I can.

In the meantime, try running the commands electrum-ltc-wrapper and electrum-ltc. I'm guessing you already tried the latter.

If none of that works, try moving the packages directory from electrum into the electrum-ltc directory, then run ./electrum-ltc.

About 1024x768, is this the actual size of your screen, or are you running a VM? I wouldn't advise running a VM because the host client can compromise the guest's security. Still better than a hot wallet, of course, but I wouldn't risk it. Most notebooks support at least 1366x768, which is fine.

Anyway, I have plans to make an HTML page aggregating all the javascript-based wallets and tools to reclaim space in the bar. This should solve your problem.

**Comment by LF76** [comment]
Obrigado pela resposta, Estêvão!!

I`m trying the ideas you posted. Indeed, in the Electrum-LTC folder, there isn't the _packages_ directory. I tried to copy it, but it seems I don't have permissions in that folder. Anyway, I guess you found the solution to this bug, by adding the missing packages there.

About the screen size, I used a VW just to test it before saving to the USB stick (so the 1024x768), but in my notebook (at 1366x768), still the icons in the borders are hidden/cut.

The HTML file is a great idea anyway to solve it, and/or the icons' size changed to half. Or, alternatively, just save the shortcuts in the Desktop, perhaps.
